app/Resources/views/themes/tastefulplanet/full/place_list.html.twig

```diff
@@ -19,8 +19,13 @@
                 </div>
             </div>
         {% else %}
-            <div class="map-container"></div>
-            {% include '@ezdesign/parts/map.html.twig' with {'contentArray': app_search_result_extract(places)} only %}
+            {% set map_container_id = 'map_container_id_' ~ random() %}
```

Comment:
Not sure you need `random` here. You will only have one map on this page. Right?

Reply:
It will be better to keep it like that, for testing purposes general usages